CHANGELOG.md

```diff
@@ -3,7 +3,11 @@
 All notable changes to this project will be documented in this file, per [the Keep a Changelog standard](http://keepachangelog.com/).
 
 ## [Unreleased] - TBD
-- Issues reported by PHPCS WPCS v2.3.0 addressed
+### Changed
+- Bump WordPress version "tested up to" 5.9 (props [@sudip-10up](https://github.com/sudip-10up) via []().
```

Comment:
Should `via []().` include a link or be removed?

Reply:
I was starting to contemplate updating the CHANGELOG.md within PRs, but yeah I don't like how the PR link is invalid until you know what the PR # is going to be so I've now stopped including this.  Will leave it on the to-do list to look at finding a GH Action that could pull our changelog and props details in a PR and auto-generate the changelog info.